src/Symfony/Component/HtmlSanitizer/HtmlSanitizerConfig.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer;
+
+use Symfony\Component\HtmlSanitizer\Reference\W3CReference;
+use Symfony\Component\HtmlSanitizer\Visitor\AttributeSanitizer\AttributeSanitizerInterface;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ */
+class HtmlSanitizerConfig
+{
+    // Elements that should be removed but their children should be retained
+    private array $blockedElements = [];
+
+    // Elements that should be retained, with their allowed attributes
+    private array $allowedElements = [];
+
+    // Links schemes that should be retained, other being dropped
+    private array $allowedLinksSchemes = ['http', 'https', 'mailto', 'tel'];
+
+    // Links hosts that should be retained (by default, all hosts are allowed)
+    private ?array $allowedLinksHosts = null;
+
+    // Should the sanitizer allow relative links (by default, they are dropped)
+    private bool $allowedLinksRelative = false;
+
+    // Image/Audio/Video schemes that should be retained, other being dropped
+    private array $allowedMediaSchemes = ['http', 'https', 'data'];
+
+    // Image/Audio/Video hosts that should be retained (by default, all hosts are allowed)
+    private ?array $allowedMediaHosts = null;
+
+    // Should the sanitizer allow relative media URL (by default, they are dropped)
+    private bool $allowedMediaRelative = false;
+
+    // Should the URL in the sanitized document be transformed to HTTPS if they are using HTTP
+    private bool $forceHttpsUrls = false;
+
+    // Sanitizers that should be applied to specific attributes in addition to standard sanitation
```

Comment:
The property description should be in the doc comment of the property rather than in a separate comment